### PR TITLE
Add new images and captions for frame

### PR DESCRIPTION
**Pull request by hanywang2**
## Summary

- Add captions as a separate section on the frame docs
- Remove forest and mountain images from local host in favor of a CDN hosted image

CleanShot 2023-01-20 at 20 47 36@2x
CleanShot 2023-01-20 at 20 47 28@2x

### CODE REVIEW

**Review by tobias-mintlify — APPROVED** [review]

File: docs/writing-content/image.mdx [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: "Images"
 description: "Create images using markdown or embeds"
 ---
 
-<img class="rounded-md" src="/docs/img/forrest.jpg" />
+<img class="rounded-md" src="https://mintlify-assets.b-cdn.net/forrest.jpg" />
```
Comment (benjamin-shen) [inline docs/writing-content/image.mdx:6]: https://grammarhow.com/forrest-or-forest/

File: docs/components/frame.mdx [hunk 2]
```diff
@@ -5,14 +5,28 @@ description: "Use the Frame component to wrap images or other components in a co
...
-  <img class="rounded-md" src="/img/mountain.jpg" />
+<Frame>
+  <img class="rounded-md" src="https://mintlify-assets.b-cdn.net/yellowstone.jpeg" />
+</Frame>
+
+## Captions
+
+You can add additional context to an image using the optional `caption` prop.
+
+<Frame caption="Yosemite National Park is visited by over 3.5 million people every year">
```
Comment (benjamin-shen) [inline docs/components/frame.mdx:16]: This caption doesn't show up in your PR description screenshot 🤔

Comment (hanywang2) [reply docs/components/frame.mdx:16]: Yea I thought it makes sense when it's a different example cause the images are also different

Comment (hanywang2) [reply docs/components/frame.mdx:16]: I know I don't have the latest CLI yet. Will upgrade later

**Pull request merged**